src/ResourceManager/Network/Commands.Network/ApplicationGateway/BackendAddressPool/AzureApplicationGatewayBackendAddressPoolBase.cs

```diff
@@ -30,12 +30,12 @@ public class AzureApplicationGatewayBackendAddressPoolBase : NetworkBaseCmdlet
         [Parameter(
                 HelpMessage = "IP addresses of application gateway backend servers")]
         [ValidateNotNullOrEmpty]
-        public List<string> BackendIPAddresses { get; set; }
+        public string[] BackendIPAddresses { get; set; }
```

Comment:
@EvgenyAgafonchikov I'm not sure if all of the markdown files were updated properly. It seems a lot of the type changes from list to array are not updated properly for the corresponding parameters. Would you make sure that the help files have been updated properly?

Reply:
@EvgenyAgafonchikov the INPUTS section is not updated using `platyPS` -- it's something that would need to be updated manually. You can do that if you'd like in this PR, but I'm more concerned about the parameter types in the markdown help being updated.

Reply:
updated INPUTS section too